### DISCUSSION TITLE
Avoid the re-render

### DISCUSSION THREAD

**Discussion by testpushhydra** [SBoudrias/Inquirer.js#1356] [Q&A]
I want to add a feature to avoid the rerendering of the results so it just stays a single line per question/answer.

I know where in the code to put the feature:

`
  onEnd(state) {
    ...

    // Re-render prompt
    
    if (!noRerender) {
      this.render();
    }
}
`

How do I get involved to add this?  Are you okay with this?  I did this directly in the node_modules code files but that is way to dirty and uncomfortable.  Please advise.

**Comment by SBoudrias** [comment]
Hi, I'd be happy to consider it, but I don't really understand your request. What is the current behaviour, and what is the desired behaviour?

Also, do not you're using the old version of Inquirer (see new release announcement here #1214.) I'm happy to consider, but I'm trying to keep API changes/extension to a minimal on the old release.

**Reply by testpushhydra** [reply]
Thanks for responding Simon.  The version I'm using replays the question and answer after the answer is entered.  In my case, I'm asking more than say 20 questions.  I prefer to have just the one line for each and not have it replay the answer.  Example:

![Inquirer](https://github.com/SBoudrias/Inquirer.js/assets/71581705/f56bbe37-851e-4f7d-81ab-aad4dec1ee82)

**Reply by SBoudrias** [reply]
Did you try using the `clearPromptOnDone` option?
